clients/src/test/java/org/apache/kafka/common/utils/UtilsTest.java

```diff
@@ -854,7 +854,7 @@ public void shouldThrowOnInvalidDateFormatOrNullTimestamp() {
 
     private void checkExceptionForGetDateTimeMethod(Executable executable) {
         assertTrue(assertThrows(ParseException.class, executable)
-            .getMessage().contains("Unparseable date"));
+            .getMessage().contains("Unparsable date"));
```

Comment:
we have some test failures associated with this: https://ci-builds.apache.org/job/Kafka/job/kafka-pr/job/PR-13884/3/testReport/junit/org.apache.kafka.common.utils/UtilsTest/Build___JDK_11_and_Scala_2_13___shouldThrowOnInvalidDateFormatOrNullTimestamp__/ 

Please fix the source code along with the test validation.

Reply:
fixed!